app/models/ability.rb

```diff
@@ -41,9 +41,12 @@ def initialize(user)
       can :change_admin, :administrator
       can :change_editor, :administrator
       can :change_basic, :administrator
+      can :sort, :administrator
```

Comment:
change_admin, change_editor, change_basic seem to be still needed, because when I took those lines away, I was no longer able to change permissions of users.